### PR TITLE
[staging-17.0] Merge master to staging-17.0 up to d095ab912

### PR DESCRIPTION
**Pull request by darcywong00**
6 of x Merging master to staging-17.0 up to d095ab912

### CODE REVIEW

File: experimental/r/runeboard/source/runeboard.kps [hunk 1]
```diff
@@ -126,7 +126,8 @@ by which fonts are installed on that machine.</Description>
     <Keyboard>
       <Name>Runes</Name>
       <ID>runeboard</ID>
-      <Version>2.1</Version>
+      <Version></Version>
```
Comment (mcdurdin) [inline experimental/r/runeboard/source/runeboard.kps:129]: Is version number meant to disappear here?

File: release/fv/fv_kwadacha_tsekene/fv_kwadacha_tsekene.keyboard_info
Comment (mcdurdin) [inline release/fv/fv_kwadacha_tsekene/fv_kwadacha_tsekene.keyboard_info:1]: This file should not be in the PR. Description needs to migrate to the .kps.

**Review by mcdurdin — CHANGES_REQUESTED** [review]

File: experimental/r/runeboard/source/runeboard.kps (hunk 1 above)
Comment (darcywong00) [reply experimental/r/runeboard/source/runeboard.kps:129]: Looks like version number was removed in #2486

**Comment by mcdurdin** [comment]
Build failure:

```
fv_kwadacha_tsekene.kpj - info KM05002: Building release/fv/fv_kwadacha_tsekene/fv_kwadacha_tsekene.kpj
fv_kwadacha_tsekene.kpj - error KM0900A: No license for the keyboard was found. MIT license is required for publication to Keyman keyboards repository.
```

File: release/fv/fv_kwadacha_tsekene/fv_kwadacha_tsekene.keyboard_info
Comment (darcywong00) [reply release/fv/fv_kwadacha_tsekene/fv_kwadacha_tsekene.keyboard_info:1]: k - will remove on the next PR. There's a descrepency in language tag
kps has `sek-Latn`.
keyboard_info has `sek`

Comment (darcywong00) [reply release/fv/fv_kwadacha_tsekene/fv_kwadacha_tsekene.keyboard_info:1]: Description and build failure below resolved in #2774. I don't think I  resolved the `sek` language tag though.
for now, keeping kps `sek-Latn`

**Review by mcdurdin — APPROVED** [review]
lgtm with patches in #2774.

**Pull request merged**